tdwii_plus_examples/rtbdi_creator/mainbdiwidget.py

```diff
@@ -95,6 +97,33 @@ def _plan_button_clicked(self):
             self.plan_path = path.parent
         # print("Plan Button Clicked")
 
+    @Slot()
+    def _store_plan_button_clicked(self):
+        store_scu = StoreSCU(self.ae_title, self.ui.line_edit_move_scp_ae_title.text())
+        # always re-load.  Not performant, but safe choice.
+        # a more performant approach would be to force a re-load every time a plan is selected.
+        # that's not really necessary unless the plan has to be (C-)stored
+        plan = load_plan(self.ui.lineedit_plan_selector.text())
+        iods = list()
+        iods.append(plan)
+        success = store_scu.store(iods=iods)
```

Comment:
It took me a while to think `iods=iods` then realized `<para>=<value>`.
Should equivalent to `success = store_scu.store(iods=[plan])`

Reply:
I originally wrote this a store_scu.store(list(plan)) and that didn't work (not sure why).
the <para>=<value> with the names being the same is a relatively common thing in python.